### PR TITLE
Override cdvMinSdkVersion if it's lower than needed

### PR DESCRIPTION
**Pull request by alsorokin**
This is a fix for https://github.com/phonegap/phonegap-plugin-barcodescanner/issues/337

Repro (cordova@6.4.0):
```
cordova create crosswalk-problem
cd crosswalk-problem
cordova platform add android
cordova plugin add https://github.com/phonegap/phonegap-plugin-barcodescanner
cordova plugin add https://github.com/crosswalk-project/cordova-plugin-crosswalk-webview
cordova compile
```

### CODE REVIEW

File: platforms/android/xwalk.gradle [hunk 1]
```diff
@@ -25,6 +25,13 @@ def SHARED_ARTIFACT_ID = "xwalk_shared_library:"
 def EMBEDD_ARTIFACT_ID = "xwalk_core_library:"
 def CANARY_ARTIFACT_ID = "xwalk_core_library_canary:"
 def BIT_64 = ":64bit@aar"
+def MIN_SDK_VERSION = 16
+
+if (cdvMinSdkVersion == null) {
+    ext.cdvMinSdkVersion = MIN_SDK_VERSION;
+} else if (cdvMinSdkVersion.toInteger() < MIN_SDK_VERSION) {
+    ext.cdvMinSdkVersion = MIN_SDK_VERSION;
```
Comment (fujunwei) [inline platforms/android/xwalk.gradle]: It has been configured to 16 by default. Maybe it need to be announced if the cdvMinSdkVersion is modified lower to 16.

> if (cdvMinSdkVersion == null) {
    def minSdk = getConfigPreference("android-minSdkVersion");
    ext.cdvMinSdkVersion =  minSdk && minSdk.toInteger() > 14 ? minSdk : 14;
}

**Comment by alsorokin** [comment]
@fujunwei I think I've fixed it now, please take a look

**Comment by baleboy** [comment]
Minor nit, your PR summary still talks about version 16

**Comment by alsorokin** [comment]
@baleboy fixed :)

**Comment by alsorokin** [comment]
Friendly ping @fujunwei

**Comment by fujunwei** [comment]
Merging, thanks.

**Pull request merged**